Pipfile

```diff
@@ -15,3 +15,5 @@ gcovr = "*"
 coverage = "*"
 
 [packages]
+pandas = "*"
```

Comment:
Can these two packages be shifted to dev dependencies?

Since they are just required for testing, it can't be dependencies for using the package.

Reply:
good point, shall I just move it up to line 16 then (under [dev-packages])?

Reply:
yeah